### DISCUSSION TITLE
[2023/05/16] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#257] [General]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/256 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by chloe463** [comment]
## Node.js の `--loader` について調べたことを共有する

- swc-node の修正をするときに調べた
- `node --loader @swc-node/register/esm  src/index.mts`  のように使う
- 普通に Node.js のスクリプトを実行するように TS ファイルを実行できる
- loader に指定したスクリプトで path の解決と、ファイル読み込み(とトランスパイル) を実行する
- https://github.com/nodejs/loaders
- https://nodejs.org/api/esm.html#loaders

**Comment by chloe463** [comment]
Next.js の parallel route and intersection の話をした。プロフィールとかストーリーとかで使えそう。

来週のテーマ: Google IO
